byzcoinx/byzcoinx.go

```diff
@@ -315,6 +320,18 @@ func InitBFTCoSiProtocol(suite *pairing.SuiteBn256, c *onet.Context, vf, ack pro
 	return nil
 }
 
+// InitBDNCoSiProtocol creates and registers the protocols required to run
+// BFTCoSi to the context c over the BDN signature scheme
+func InitBDNCoSiProtocol(suite *pairing.SuiteBn256, c *onet.Context, vf, ack protocol.VerificationFn, protoName string) error {
```

Comment:
nit: do we have a convention on the capitalisation of acronyms? in this file we have both `BdnCoSi` and `BDNCoSi`

Reply:
I have a preference for camel-cased even if it's an acronym. It's slightly more readable.